src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,186 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> head;
+    private Node<T> tail;
+
+    class Node<E> {
+        private E item;
+        private Node<E> next;
+        private Node<E> prev;
+
+        Node(Node<E> prevElement, E element, Node<E> nextElement) {
+            prev = prevElement;
+            item = element;
+            next = nextElement;
+        }
+    }
```

Comment:
In my opinion this class should be static, and be at the end of your LinkedList class.